tests/__init__.py

```diff
@@ -0,0 +1,33 @@
+"""
+Tests of the pytest-repo-health plugin itself (including the fixtures it provides).
+"""
+from pathlib import Path
+
+PYTEST_INI = """
+[pytest]
+addopts = -vv --repo-health --repo-health-path {checks_path} --repo-path {repo_path}
```

Comment:
should the -vv be -v?

Reply:
That was just for debugging, I'll remove it.  Adding a changelog entry and bumping the version while I'm at it.